src/components/navbar/Navbar.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './navbar.css';
+
+const Navbar = () => (
+  <div className="nav-container">
+    <nav className="nav-bar">
+      <div>
+        <ul>
+          <li>
+            <h2>Book Store</h2>
+          </li>
+          <li>
+            <a href="/">Home</a>
+          </li>
+          <li>
+            <a href="/categories">Categories</a>
+          </li>
```

Comment:
As specified in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/3954da2af5b97dd8269f92f236d92c29844679ad/bookstore/project_init_project_wth_comps_v2_0.md?plain=1#L29), kindly use React's `<Link>` component instead of the conventional anchor tag.

Currently, the website reloads on navigating from one route to another and this defeats the purpose of having a single page application.